wrappers/rust/icicle-core/src/ntt/mod.rs

```diff
@@ -38,6 +49,22 @@ pub enum Ordering {
     kNR,
     kRN,
     kRR,
+    kNM,
+    kMN,
+}
+
+///Which NTT algorithm to use. options are:
+///- Auto: implementation selects autoamtically based on heuristic
```

Comment:
autoamtically -> automatically

Reply:
Also maybe worth mentioning that Auto is a good default choice and should be used in most cases?

Reply:
Thanks. Will fix typo and mention that.